sdk/keyvault/keyvault-keys/test/public/crypto.spec.ts

```diff
@@ -60,6 +60,19 @@ describe("CryptographyClient (all decrypts happen remotely)", () => {
       assert.equal(text, decryptedText);
     });
 
+    it("encrypt with additional parameters returns them", async function() {
+      const text = this.test!.title;
+      const encryptResult = await cryptoClient.encrypt("RSA1_5", stringToUint8Array(text), {
+        iv: stringToUint8Array("iv"),
+        tag: stringToUint8Array("tag"),
+        additionalAuthenticatedData: stringToUint8Array("aad")
+      });
+      console.log("encryptResult", encryptResult);
+      assert.equal(uint8ArrayToString(encryptResult.iv!), "iv");
```

Comment:
I don't know if this makes sense! Since `iv` is specific for some algorithms (none of them being RSA, that I'm aware of).

Reply:
Yea we have a problem right now where unfortunately we can't run AES algorithms against non MHSM keyvault (I talked to Ben from engsys and we're working on a solution). Still, if the user passes in an iv we should respect that right?

Reply:
I think the algorithms that don't use iv should not do anything with it 🤔 we use our tests to showcase how to use our code. Perhaps something mocked on the internal tests? test/internal/*